### PR TITLE
@anyone => test jam

### PR DESCRIPTION
**Pull request by orta**
TheTest Jam blog post.

Thanks @modocache / @lazerwalker

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ _site
 .sass-cache
 _gh-pages
 .DS_Store
+.Ulysses-Group.plist
```
Comment (kylef) [inline .gitignore]: What's this? Doesn't seem project specific, probably belongs in global git ignore?

File: _posts/2015-02-21-Test-Jammin.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+*TLDR/Elevator Pitch*: If the community is going to move towards Swift, then let’s make sure that our Objective-C is in order by adding tests to established libraries.
+
+Find out more below.
+
+<!-- more -->
+
+Date: A weekend in late April / early May
+Locations: NYC / SF / + More
+
+We have picked out 200 pods without tests, or with very low test coverage, ordered by [popularity][2]. These are assigned randomly to interested contributors via an IRC bot. They then will aim to:
```
Comment (kylef) [inline _posts/2015-02-21-Test-Jammin.markdown]: I think we're moving very quickly here. We should probably explain what IRC is first.

Comment (orta) [inline _posts/2015-02-21-Test-Jammin.markdown]: removed all references to IRC :star:

**Comment by segiddins** [comment]
I need to check that we can host. @timanglade, do you think we/I could do this?

**Comment by orta** [comment]
Shipping, happy to make changes as we get more detail around dates etc too

**Comment by timanglade** [comment]
Oh for sure we’re in. We don’t have much planned in late April early May. Count @realm in.

**Comment by orta** [comment]
:tada:

**Pull request merged**